lib.php

```diff
@@ -338,6 +338,61 @@ function sortvoting_get_sortvoting($sortvotingid) {
     return false;
 }
 
+/**
+ * Get all the responses for a sortvoting.
+ *
+ * @param stdClass $sortvoting
+ * @param stdClass $cm
+ * @param int $groupmode
+ * @param bool $onlyactive Whether to get response data for active users only.
+ * @return array
+ */
+function sortvoting_get_response_data(stdClass $sortvoting, stdClass $cm, int $groupmode, bool $onlyactive = true): array {
+    global $CFG, $USER, $DB;
+
+    $context = context_module::instance($cm->id);
+
+    // Get the current group.
+    if ($groupmode > 0) {
+        $currentgroup = groups_get_activity_group($cm);
+    } else {
+        $currentgroup = 0;
+    }
```

Comment:
You can simplify all this (from L356-360) by just calling `$currentgroup = groups_get_activity_group($cm);` and passing the result to `get_enrolled_users`